guidelines/content-guidelines/README.md

```diff
@@ -7,7 +7,7 @@ These guidelines provide rules and tips to all who contribute content to the Kym
 Read about:
 
 - The [content strategy](./content-strategy.md) for the Kyma documentation. Learn about the obligatory and optional document types.
-- [Formatting](./formatting-and-style.md), [style, and standards](./style-and-standards.md) that apply to the content created throughout the Kyma organization.
+- [Formatting](./formatting.md), [style, and terminology](./style-and-terminology.md) that apply to the content created throughout the Kyma organization.
```

Comment:
It may look better without the comma, what do you think?
```suggestion
- [Formatting](./formatting.md), [style and terminology](./style-and-terminology.md) that apply to the content created throughout the Kyma organization.
```